### PR TITLE
[benchmark tools] Update mobilenet models

### PR DESCRIPTION
**Pull request by Linchenn**
As discussed in the spreadsheet, we will supports alpha variable for mobilenet V2 and V3, so this PR:
1. Supports `075 and 100 alphas` and `small and large architectures` for mobilenet v3.
2. Supports `050, 075 and 100 alphas` for mobilenet v2.
3. Updates README.

To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

- - -
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/6729)

### CODE REVIEW

**Review by Linchenn — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/6729)* status: 0 of 1 approvals obtained (waiting on @lina128 and @qjia7)

___
*[`e2e/benchmarks/local-benchmark/README.md` line 70 at r2](https://reviewable.io/reviews/tensorflow/tfjs/6729#-N8pbb_Y117fN8QgXEeA:-N8pbb_Y117fN8QgXEeB:bifv61b) ([raw file](https://github.com/tensorflow/tfjs/blob/4340da3bd7ccdfe396466b326127efefac8a48e5/e2e/benchmarks/local-benchmark/README.md#L70)):*
> ```Markdown
> It's easy to set up a web server to host benchmarks and run against them via e2e/benchmarks/local-benchmark/index.html. You can manually specify the optional url parameters as needed. Here are the list of supported url parameters:
> 
> * Model related parameters:
> ```

Just added `alpha` and separated the parameters into two categories for easy understanding.

File: e2e/benchmarks/local-benchmark/index.html [hunk 1]
```diff
@@ -198,7 +199,7 @@ <h2>TensorFlow.js Model Benchmark</h2>
       numWarmups: warmupTimes,
       numRuns: runTimes,
       numProfiles: profileTimes,
-      benchmark: 'mobilenet_v2',
+      benchmark: 'mobilenet_v3',
```
Comment (qjia7) [inline e2e/benchmarks/local-benchmark/index.html:202]: Forget to change to `MobileNetV3`?

**Review by qjia7 — APPROVED** [review]
LGTM with one change.

Nice work!

File: e2e/benchmarks/local-benchmark/index.html (hunk 1 above)
Comment (Linchenn) [reply e2e/benchmarks/local-benchmark/index.html]: Done. Thank you!

**Review by pyu10055 — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/6729)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @lina128, @Linchenn, and @qjia7)

___
*[`e2e/benchmarks/local-benchmark/index.html` line 202 at r3](https://reviewable.io/reviews/tensorflow/tfjs/6729#-N8pa1a_qzL7W3Qo2-xD-r3-202:-N8z3NXZ4Y2UpfyEnNJk:bsqk344) ([raw file](https://github.com/tensorflow/tfjs/blob/a0b44e40b8571313124242d490cda31b4ea393e2/e2e/benchmarks/local-benchmark/index.html#L202)):*
<details><summary><i>Previously, Linchenn wrote…</i></summary><blockquote>

Done. Thank you!
</blockquote></details>

does mobilenet_v2 have architecture and alpha configurations?
___
*[`e2e/benchmarks/local-benchmark/README.md` line 78 at r1](https://reviewable.io/reviews/tensorflow/tfjs/6729#-N8z2Up38l9BjCDIexaH:-N8z2Up38l9BjCDIexaI:b-y7meoo) ([raw file](https://github.com/tensorflow/tfjs/blob/3f772ae1d9454315d1a667750b0084c57245cf71/e2e/benchmarks/local-benchmark/README.md#L78)):*
> ```Markdown
>   <b>inputType</b>: same as inputTypes<br>
>   <b>modelUrl</b>: same as modelUrl, for custom models only<br>
>   <b>${InputeName}Shape</b>: the input shape array, separated by comma, for     custom models only.  For example, bodypix's [graph model](https://storage.    googleapis.com/tfjs-models/savedmodel/bodypix/mobilenet/float/075/    model-stride16.json) has an input named sub_2, then users could add     '`sub_2Shape=1,1,1,3`' in the URL to populate its shape.<br>
> ```

this url looks weird, please double check
___
*[`e2e/benchmarks/local-benchmark/README.md` line 70 at r2](https://reviewable.io/reviews/tensorflow/tfjs/6729#-N8pbb_Y117fN8QgXEeA:-N8z2e0Z2_XGDejA6YYz:b-9v9dyi) ([raw file](https://github.com/tensorflow/tfjs/blob/4340da3bd7ccdfe396466b326127efefac8a48e5/e2e/benchmarks/local-benchmark/README.md#L70)):*
<details><summary><i>Previously, Linchenn wrote…</i></summary><blockquote>

Just added `alpha` and separated the parameters into two categories for easy understanding.
</blockquote></details>

architecture and alpha are related to certain type of models (mobilenet, maybe posenet?), might be good to point it out.

**Review by Linchenn — COMMENTED** [review]
Thank you!

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/6729#-:-N8zCl3X2AjuiMRTmUkQ:b-9755fn)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @lina128, @pyu10055, and @qjia7)

___
*[`e2e/benchmarks/local-benchmark/index.html` line 202 at r3](https://reviewable.io/reviews/tensorflow/tfjs/6729#-N8pa1a_qzL7W3Qo2-xD-r3-202:-N8zBZ-OCMReCLY-wx5S:bif90nr) ([raw file](https://github.com/tensorflow/tfjs/blob/a0b44e40b8571313124242d490cda31b4ea393e2/e2e/benchmarks/local-benchmark/index.html#L202)):*
<details><summary><i>Previously, pyu10055 (Ping Yu) wrote…</i></summary><blockquote>

does mobilenet_v2 have architecture and alpha configurations?
</blockquote></details>

Yes, I added it here https://github.com/tensorflow/tfjs/pull/6729/files#diff-4c36570d490a92c2848da5b19e986e30c6663f7d929d58f8fd342534e6e793b6R121.
___
*[`e2e/benchmarks/local-benchmark/README.md` line 78 at r1](https://reviewable.io/reviews/tensorflow/tfjs/6729#-N8z2Up38l9BjCDIexaH:-N8zBnG86Wh9pn0kJPE6:bpqd6ax) ([raw file](https://github.com/tensorflow/tfjs/blob/3f772ae1d9454315d1a667750b0084c57245cf71/e2e/benchmarks/local-benchmark/README.md#L78)):*
<details><summary><i>Previously, pyu10055 (Ping Yu) wrote…</i></summary><blockquote>

this url looks weird, please double check
</blockquote></details>

This is updated and you may have to click `show full diff`:

___
*[`e2e/benchmarks/local-benchmark/README.md` line 70 at r2](https://reviewable.io/reviews/tensorflow/tfjs/6729#-N8pbb_Y117fN8QgXEeA:-N8zCbdR3eVumXwoXJgG:b-o7vuhc) ([raw file](https://github.com/tensorflow/tfjs/blob/4340da3bd7ccdfe396466b326127efefac8a48e5/e2e/benchmarks/local-benchmark/README.md#L70)):*
<details><summary><i>Previously, pyu10055 (Ping Yu) wrote…</i></summary><blockquote>

architecture and alpha are related to certain type of models (mobilenet, maybe posenet?), might be good to point it out.
</blockquote></details>

Good catch! Done.

**Review by Linchenn — COMMENTED** [review]
As discussed with Ping, I moved `alpha` model parameter into `architecture` model parameter, because `alpha` is a special parameter for mobilenet only and it could be considered a part of `architecture` definition. 

@qjia7 FYI.

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/6729#-:-N8zKE6F6F1G_Gdsax_a:b-gk47gf)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @lina128, @pyu10055, and @qjia7)

**Review by lina128 — APPROVED** [review]
LGTM

**Review by qjia7 — APPROVED** [review]
>As discussed with Ping, I moved alpha model parameter into architecture model parameter, because alpha is a special parameter for mobilenet only and it could be considered a part of architecture definition.

Still LGTM. Thank you!

**Review by pyu10055 — APPROVED** [review]
> Reviewed 3 of 3 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/6729)* status: :shipit: complete! 3 of 1 approvals obtained (waiting on @pyu10055 and @qjia7)

**Pull request merged**